### PR TITLE
Template integration

### PR DESCRIPTION
**Pull request by kartikey-thoughtwin**
- Template integration with Notes APIs [Done]

### CODE REVIEW

File: .gitignore
Comment (shubhamvisaretw) [inline .gitignore:1]: Ok.

File: app/__init__.py
Comment (shubhamvisaretw) [inline app/__init__.py:1]: Ok

File: app/models/note.py
Comment (shubhamvisaretw) [inline app/models/note.py:1]: "created_at": self.created_at.isoformat() if self.created_at else None

must not return None in any case. created timing must be set on creation only.

Comment (kartikey-thoughtwin) [reply app/models/note.py:1]: Sure, I'll change

**Pull request merged**